apps/src/code-studio/pd/regional_partner_contact/RegionalPartnerContact.jsx

```diff
@@ -109,6 +109,14 @@ class RegionalPartnerContactComponent extends FormComponent {
           })
         }
         {
+          this.buildButtonsFromOptions({
+            name: 'program',
+            label: 'What program are you interested in?',
```

Comment:
`Which programs are you interested in?` works better since this is a multiselect